### PR TITLE
captain ID spawns in the hands of the highest-ranking head of staff if present

### PR DESCRIPTION
**Pull request by I-SNORT-XENOMORPH-ACID-OCCASIONALLY**
:cl:
 * tweak: captain ID spawns in the hands of the highest-ranking head of staff if present

### CODE REVIEW

**Comment by ShiftyRail** [comment]
Maybe with a small message telling them they're the captain and so on.

File: __DEFINES/global.dm [hunk 1]
```diff
@@ -469,3 +469,13 @@ var/list/variables_not_to_be_copied = list(
 	"on_explode","on_projectile","in_chamber","power_supply","contents",
 	"x","y","z"
 )
+
+// http://ss13.moe/wiki/index.php/Chain_of_Command
+var/list/chain_of_command = list(
+	"Captain" = 1,
+	"Head of Personnel" = 2,
```
Comment (ShiftyRail) [inline __DEFINES/global.dm:476]: I would personally put HoPs at the same level of HoSes but I'm slightly biased

Comment (I-SNORT-XENOMORPH-ACID-OCCASIONALLY) [reply __DEFINES/global.dm:476]: http://ss13.moe/wiki/index.php/Chain_of_Command

Comment (ShiftyRail) [reply __DEFINES/global.dm:476]: Outdated (literal) bay article that doesn't really take into account the concrete reality of the server if you ask me.

Comment (I-SNORT-XENOMORPH-ACID-OCCASIONALLY) [reply __DEFINES/global.dm:476]: i don't

**Comment by ghost** [comment]
spare rushers blown the FUCK out

**Comment by Chazach** [comment]
muh battle royale

**Comment by Irradiation** [comment]
Where have I seen this PR before?

**Comment by Commandersand** [comment]
image

**Comment by ghost** [comment]
image

**Comment by ShiftyRail** [comment]
Also a few comments worth taking into account based on [21472], which did it in a much better way, code-wise.

comment 1

comment 2


In essence :

- there is absolutely no standards for heads of staff. I fear some non-willing CMO or non-willing RD taking the spare would result in them putting it in their backpack and forgetting it, resulting in an even worst situation than if a greyshirt secures it.

- CMO, RD, and CEs aren't implanted nor have reduced antag chances. This is not as silly as it sounds.

**Comment by Chazach** [comment]
pros: i dont have to join the rush for the thanos glove all the time
cons: the cultist RD/CMO/CE gets the thanos glove by default
overall, its a no from me

**Comment by ShiftyRail** [comment]
The alternative solution is to make the RD, CMO, and CE start implanted with reduced antag chances and the basic expectations to do a minimum amount of captain work if they are chosen
E. g. doing access changes, cargo orders or promoting people to QM

**Comment by SonixApache** [comment]
#21472 didn't pass, why should this

**Comment by Accipitermaris** [comment]
should re-enable the config that makes someone random captain at roundstart if there’s no captain & let players enable or disable their “chance” for random forced captaincy in job prefs. (optional: give some cosmetic reward for enabling random forced captaincy.)
if no one has their random captaincy enabled in job prefs everyone spawns with all access

**Comment by Accipitermaris** [comment]
@sonixapache that had a (barely) positive reception and i say that as someone who does not want its followup (this pr)

**Comment by iFlashYou** [comment]
But fuck the disk and pinpointer

**Comment by utilitas-** [comment]
@Accipitermaris We already have that. It's called *job preferences*. It's where you can set different jobs to be picked based on priority, *fancy that.* All head jobs automatically force someone to be them even if they are just on low, ergo nobody competent plays them. Stop needlessly complicating everything.

**Comment by Accipitermaris** [comment]
its not so much a needless complexity so much as it is a config that we disabled because it would force inexperienced people into captaincy
that's why there's a "captaincy is not forced upon anyone" line at the start of every round

**Comment by ShiftyRail** [comment]
>All head jobs automatically force someone to be them even if they are just on low, 

Ι have a character with HoS on medium and Sec Off on high and I only roll HoS if there is already a few officers around.

**Comment by Caboozles** [comment]
How about instead we have a five-fifteen minute grace period before the spare appears if there is no roundstart Captain?

That way there isn’t a need to speedrun, but the spectre of someone grabbing it is still there if a head or security doesn’t respond appropriately.

**Comment by utilitas-** [comment]
How about you fix captain and make it so people actually *want to play captain* instead of spreading his job around to everyone else, like some kind of retarded hot potato?

File: __DEFINES/global.dm (hunk 1 above)
Comment (jknpj) [reply __DEFINES/global.dm:476]: That's just shifty's headcanon

**Comment by utilitas-** [comment]
In fact, here; Suggestions that would make people actually want to play heads, so we don't have to make retarded changes that'd inadvertently make heads even worse to play for everyone involved:
- Give heads goodies that only they may get. That would be though head uplinks, not too dissimilar from traitor uplinks. An unlockable menu with a choice of departmental items to be bought with telecrystals, that may improve the lives of both them and their coworkers. Say, for the CMO: A choice between advanced medical bots, one-time full upgrade to any one machine, chemist's set (peridaxon, rezadone and the works), random nofail power injector, et cetera.
- The Captain should get some goodies to distinguish him from the others, goodies that might be quite powerful in the right hands. Maybe even a little metagamey. Specifically, the captain should get an old-style status summary after the ten minute mark that has a good chance of either (at the time of printing): list the name of a syndicate agent in the crew (where applicable), randomly list one of the antagonists currently in existence (by body count), or list the department with the most antagonists by percent. Might be powerful in the right hands, or it might not, but it *needs* a small failure chance to prevent total metagay. I believe the old status summary had a 1/5 chance of being completely wrong, and a 20% chance for this shouldn't be amiss, either.
- A visual threat indicator which displays both the current danger and the current unspent threat. Mostly decoration- there is not much you can ascertain from it apart from sudden jumps or the general amount of gunk you should expect.
- A portable CentComm communicator. Talk to CentComm and make requests anywhere, anytime- even in total blackouts and brownouts. Could be used to summon an ERT, even.
- Pinpointer conversion kit. Now it no longer points to just the disk, but can be changed to point towards: The nearest communication console, the position of the station's nuke (come on, those things ought to have trackers), the position of the AI or the position of a custom bug (appended to the conversion kit- can be attached to any object or movable structure, like the detective's camera bug). Now you can always know who's stolen your fuckin' laser gun.
- Superloyalty implant kit. Greytide implant on drugs. High cost, high payoff, and likely only one charge. Whoever gets implanted with this bad boy loses their antag status and intrinsically antagonist-related skills (that means cult tats and abilities, though not spells because those can be gained by non-wizards) *no matter what it may be* and gets the single objective of protecting whoever implanted them.
- Emergency shuttle accelerator. What it says on the tin. Instead of 10 minutes, it takes only around ~5 +-1 minutes for the shuttle to arrive. Recalls disabled under 5 minutes, as always. Afterwards, it takes only two head cards to authenticate shortening to 10 seconds. Abuse should be penalized accordingly.
- Delta-level authentication card. Enables elevating to the delta code threat level, which allows execution of subversive elements on sight, the summoning of a deathsquid and gives the captain the nuke code. As always, it requires the cooperation of at least two heads of staff.

Right now, I cannot think of anything the captain should get that isn't all about *dealing with fuckups*, but that is effectively all a captain does. Perhaps a one-time emergency relief shuttle sent from CentComm with supplies and even possible "expert" crew (acting like a respawn for ghosts) aboard?

**Comment by ghost** [comment]
Head roles should absolutely 100% not get free validhunting tools due to the cancer it would attract. If you want to give heads bonuses, the CMO suggestions of things that give a slight jumpstart are far more acceptable. Stuff that helps the department get set up and running smoother would be a good way to get people to play (and maybe even respect) heads a bit more.

**Comment by Kurfursten** [comment]
not enough support

**Pull request closed without merging**